overview.md

```diff
@@ -0,0 +1,180 @@
+# STAC Overview
+
+There are three component specifications that together make up the core SpatioTemporal Asset Catalog specification.
+Each can be used alone, but they work best in concert with one another. The [STAC API specification](https://github.com/radiantearth/stac-api-spec) 
+builds on top of that core, but is out of scope for this overview. Items are the core atomic unit, representing a 
+single spatiotemporal asset as GeoJSON so it can be searched. Catalogs are structural elements, used to group Items
```

Comment:
As a newcomer, I'd ask myself: What's an asset?

Reply:
I have a section just below called 'What is a SpatioTemporal Asset'. I guess maybe link to it from here? Or put it before the item overview?

Reply:
Yes, a link would be good at that point.